common/djangoapps/student/views/dashboard.py

```diff
@@ -749,7 +750,8 @@ def student_dashboard(request):  # lint-amnesty, pylint: disable=too-many-statem
         redirect_message = ''
 
     valid_verification_statuses = ['approved', 'must_reverify', 'pending', 'expired']
-    display_sidebar_on_dashboard = verification_status['status'] in valid_verification_statuses and \
+    display_sidebar_on_dashboard = not is_integrity_signature_enabled(None) and \
```

Comment:
This is where I am wondering if we can incorporate the enrollments of this user.
I believe in this function, you should already have the enrollments of this user. Can we check if any of the course had the integrity_signature enabled? Should we only turn off the dashboard panel when all of the enrolled course has the integrity check off?
Or this is already the case? The integrity_signature check will not return true with `None` only when the flag is turned on for all courses?